requirements.txt

```diff
@@ -0,0 +1,6 @@
+celery>=3.1
+Django>=1.8,<1.10
+python-dateutil
+
+# Locking of objects using a database table
+-e git+ssh://git@github.com/vikingco/django-db-locking.git@1.1.3#egg=django-db-locking
```

Comment:
You can drop the `-e` here, some users of this library might not have a Github account.